<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(255, 170, 255)</string>
  </property>
  <widget class="QLineEdit" name="type_trat_input">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showw">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px;
background: rgb(255, 170, 127)</string>
   </property>
   <property name="text">
    <string>Показать расходы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="trat_input">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_trat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px;
background: rgb(78, 78, 115)</string>
   </property>
   <property name="text">
    <string>Введите на что пошёл расход:</string>
   </property>
  </widget>
  <widget class="QLabel" name="trat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px;
background: rgb(85, 170, 127)</string>
   </property>
   <property name="text">
    <string>Введите расход:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(170, 170, 127);
font: 20px;</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nalog">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(129, 148, 255);
font: 20px</string>
   </property>
   <property name="text">
    <string>Посмотреть мои налоги</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_log">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px;
background: rgb(114, 124, 147)</string>
   </property>
   <property name="text">
    <string>Поменять пользователя</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_kurs">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px;
background: rgb(0, 255, 127)</string>
   </property>
   <property name="text">
    <string>Перевести по курсу</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
